### ISSUE TITLE
Wiremock classpath issue with M14

### ISSUE THREAD

**Issue by mriehl** [alexarchambault/coursier#334]
Hi,

Updating from M13 to M14 my build broke. I have a minimal activator project [here](https://github.com/mriehl/coursier-test).

With `1.0.0-M13` everything works, but with `1.0.0-M14`, `./activator compile` bails out with:

```
[error] /data/home/mriehl/coursier-test/src/main/scala/com/example/Hello.scala:3: object github is not a member of package com
[error] import com.github.tomakehurst.wiremock.WireMockServer
[error]            ^
[error] one error found
[error] (compile:compileIncremental) Compilation failed
```

Looks like a classpath thing to me, and just downgrading coursier fixes it. Any ideas?
It only occurs with wiremock (as far as my dependencies are concerned) which is really weird.

**Comment by alexarchambault** [comment]
This originates from `"com.github.tomakehurst" % "wiremock" % "2.1.11"` declaring a "POM" packaging in its metadata, hence no JAR getting fetched.

That's related to https://github.com/alexarchambault/coursier/issues/278 and https://github.com/alexarchambault/coursier/issues/298, among others. The fix for those led to a bunch of regressions in `1.0.0-M13` (https://github.com/alexarchambault/coursier/issues/322, ...), hence a partial revert in `1.0.0-M14`.

As a workaround, the JAR can be explicitly pulled with
```
libraryDependencies += "com.github.tomakehurst" % "wiremock" % "2.1.11" jar()
```

I think I'll try another fix in `1.0.0-M15`.

**Comment by mriehl** [comment]
Cool, thanks for the explanation. I think it makes sense to not download jars when the packaging is `pom`, so I'll stay on `M14` and add the `jar()` pragma.

Thank you so much!
